woocommerce/woocommerce
# add cookie consent filter

for https://github.com/woocommerce/woocommerce/issues/31118

### All Submissions:


### Changes proposed in this Pull Request:

add a filter for cookie consent

Closes #31118

### How to test the changes in this Pull Request:

1. add products to cart
2. check if cookies are set/products in cart

### Other information:


### Changelog entry

> Enhancement - Add filter `woocommerce_set_cookie_enabled` to allow disabling specific cookies.

### FOR PR REVIEWER ONLY:
